app-modules/task/src/Filament/Resources/TaskResource.php

```diff
@@ -46,15 +46,15 @@ class TaskResource extends Resource
 {
     protected static ?string $model = Task::class;
 
-    protected static ?string $navigationGroup = 'Service Management';
+    protected static ?string $navigationGroup = 'Project Management';
 
-    protected static ?int $navigationSort = 90;
+    protected static ?int $navigationSort = 10;
 
-    protected static ?string $breadcrumb = 'Task Management';
+    protected static ?string $breadcrumb = 'Tasks';
```

Comment:
If we remove the breadcrumb and navigation label, does it look the same? Since these are now default values